kaldb/src/test/java/com/slack/kaldb/server/KaldbConfigTest.java

```diff
@@ -461,7 +470,9 @@ public void testEmptyJsonStringInit() throws InvalidProtocolBufferException {
   @Test
   public void testEmptyYamlStringInit()
```

Comment:
We should also add a test here that ensure that a non-index node role doesn't require a value indexer config object.